src/ui/FormWidgets/TextInput.ts

```diff
@@ -33,13 +35,20 @@ export interface ITextInputOptions {
    * A custom aria-label attribute to add to the `TextInput`'s HTML element.
    */
   ariaLabel?: string;
+  /**
+   * The name or svg source of an icon to display inside the input at its beginning.
+   *
+   * **Default:** `null`
+   */
+  prefixingIcon?: string;
```

Comment:
Would `icon` work for the name? Since there is no `suffix` icon, I don't think the name needs to be more specific.

Reply:
Good point, I was hesitant between `icon` and `prefixingIcon`.